packages/create-llama/pnpm-lock.yaml

```diff
@@ -0,0 +1,882 @@
+lockfileVersion: '6.0'
```

Comment:
Why there are the second `lock` file, I believe this monorepo should only have one such file

Reply:
good catch, i removed it